packages/babel-plugin-emotion/src/utils/arguments.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { isTaggedTemplateExpressionTranspiledByTypeScript } from './checks'
+
+export const appendExpressionToArguments = (t: *, path: *, expression: *) => {
```

Comment:
I think this replaces `appendStringToArguments` from `src/utils/strings.js` so we could we delete `appendStringToArguments`?

Reply:
`appendStringToArguments` is still used here:
https://github.com/emotion-js/emotion/blob/3459ad700b719b2c39c69e315a6fce1808443931/packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js#L149

both are very similar though so probably this could be unified somehow

Reply:
Unifying them would probably be good. The main reason I brought this up though is that I saw that appendStringToArguments wasn't being run in the tests(https://codecov.io/gh/emotion-js/emotion/pull/1656/src/packages/babel-plugin-emotion/src/utils/strings.js) which probably isn't good

Reply:
I've unified them, please take a look.